mscore/musescore.cpp

```diff
@@ -6406,6 +6406,7 @@ bool MuseScore::saveMp3(Score* score, const QString& name)
 
 bool MuseScore::saveMp3(Score* score, QIODevice* device, bool& wasCanceled)
       {
+      Q_UNUSED(wasCanceled);
```

Comment:
I'm not sure that not using this variable is a correct behavior. `wasCancelled` should be processed when exporting mp3. Maybe it is done in another method with the same name.

Reply:
We'd need @lasconic to comment on this, the code dealing with `wasCancelled` had been removed by him

Reply:
But yes, I gtess that parameter should get used to remove an obsolete mp3 when the generation gets canceled

Reply:
@lasconic: Mind to look into this `saveMp3()` issue?

Reply:
Fixed in #4327